core/translate/select.rs

```diff
@@ -396,6 +402,9 @@ pub fn translate_select(mut select: Select) -> Result<Program> {
         let _ = translate_sorter(&select, &mut program, &sort_info.unwrap(), &limit_info);
     }
 
+    if !select.exist_aggregation {
```

Comment:
Why does it matter if there are aggregations or not?

Reply:
`select count(x) from table where false;` needs to jump to `AggFinal` and produce a `0` instead of going to `Halt`